synfig-studio/org.synfig.SynfigStudio.appdata.xml.in

```diff
@@ -55,6 +55,6 @@
   <url type="translate">https://www.transifex.com/morevnaproject/synfig/</url>
   <content_rating type="oars-1.1"/>
   <releases>
-    <release version="1.4.1" date="2021-04-26"></release>
+    <release version="1.4.2" date="2021-05-14"></release>
```

Comment:
Is it okay to include this date here now?

Reply:
This is a temporary date, placed here until we know the actual release date. When release will be ready, I run another script - `version-release.sh` - and it will update the date automatically ^__^ -

https://github.com/synfig/synfig/blob/e0ad411a079db5cc545e99de286157d602144582/version-release.sh#L14-L16